qiskit/visualization/matplotlib.py

```diff
@@ -863,13 +863,13 @@ def _draw_ops(self, verbose=False):
                 elif op.name == 'initialize':
                     vec = '[%s]' % param
                     self._custom_multiqubit_gate(q_xy, wide=_iswide,
-                                                 text=op.op.label or "|psi>",
+                                                 text=getattr(op.op, 'label') or "|psi>",
```

Comment:
This fix didn't seem to work for me - I was still getting the `AttributeError`. Instead, this form worked:
```
text=getattr(op.op, 'label', '|psi>')
```
based on the [Python docs for getattr](https://docs.python.org/3/library/functions.html#getattr).

Reply:
This was done in an earlier version of the PR, see: https://github.com/Qiskit/qiskit-terra/pull/4389/commits/63d6b5938290d3d1d817460b8a1c67229b878ec5 although I can't remember the rationale for why I reverted it though

Reply:
I remember now it doesn't cover the edge case when label is defined but defined as `None`. It may not apply for initialize but it came up on the other gates which is why I reverted it